tgui/package.json

```diff
@@ -15,6 +15,10 @@
   },
   "dependencies": {
     "@babel/eslint-parser": "^7.22.15",
+    "@babel/core": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-transform-jscript": "^7.22.5",
```

Comment:
These were required for the bat to run properly now, god knows why this crept in between me making the original prettier PR and now, probably some dependabot update screwed things